scripts/standardising.py

```diff
@@ -81,42 +74,67 @@ def run_standardising(
     return standardized_tiffs
 
 
-def download_tiff_file(input_file: str, tmp_path: str) -> str:
-    """Download a tiff file and some of its sidecar files if they exist.
+def download_tiffs(files: List[str], target: str) -> List[str]:
+    """Download a tiff file and some of its sidecar files if they exist to the target dir.
 
     Args:
-        input_file: file to download
-        tmp_path: target folder to write too
+        files: links source filename to target tilename
+        target: target folder to write too
 
     Returns:
-        location of the downloaded tiff
+        linked downloaded filename to target tilename
+
+    Example:
+    ```
+    >>> download_tiff_file(("s3://elevation/SN9457_CE16_10k_0502.tif", "CE16_5000_1003"), "/tmp/")
+    ("/tmp/123456.tif", "CE16_5000_1003")
+    ```
     """
-    target_file_path = os.path.join(tmp_path, str(ulid.ULID()))
-    input_file_path = target_file_path + ".tiff"
-    get_log().info("download_tiff", path=input_file, target_path=input_file_path)
+    downloaded_files: List[str] = []
+    for file in files:
```

Comment:
is there a easy way to multithread this download? this could be 100s of <5MB files